wordpress-mobile/gutenberg-mobile
# Include test reporter for Jest

I thought it'd be good if we could see the test results without having to scroll to the step in CircleCi. This involves adding the jest-junit test reporter to our `jest.config.js` and specifying a path for the reports in Ci. Alternatively, it could be specified in the package.json when running but since we'll only use them in Ci I just took advantage of the environment option in the vi config. 

Another addition here is the `--runInBand` flag on the jest command. According to the [CircleCI docs](https://circleci.com/docs/2.0/collect-test-data/) in the Jest section: 

> When running Jest tests, please use the --runInBand flag. Without this flag, Jest will try to allocate the CPU resources of the entire virtual machine in which your job is running. Using --runInBand will force Jest to use only the virtualized build environment within the virtual machine.

To test: 

In Ci the test results should be shown in the test summary section at the top. This doesn't apply for the Check correctness tests since that's eslint and flow.
